custom-connectors/CopilotForFinanceCommunications/apiDefinition.swagger.json

```diff
@@ -1,765 +1,1376 @@
-                    }
-                  },
-                  "description": "Participants."
-                }
-              },
-              "required": [
-                "topic"
-              ]
-            }
+            "in": "query",
+            "name": "language",
+            "description": "The language code is in BCP 47 format.",
+            "type": "string"
           }
-        ]
-      },
-      "get": {
+        ],
         "responses": {
           "200": {
-            "description": "Success.",
+            "description": "The request has succeeded.",
             "schema": {
+              "required": [
+                "data"
+              ],
               "type": "object",
               "properties": {
-                "value": {
+                "data": {
                   "type": "array",
                   "items": {
-                    "type": "object",
-                    "properties": {
-                      "id": {
-                        "type": "string",
-                        "description": "Chat thread ID.",
-                        "x-ms-summary": "ID"
-                      },
-                      "topic": {
-                        "type": "string",
-                        "description": "Chat thread topic.",
-                        "x-ms-summary": "Topic"
-                      },
-                      "lastMessageReceivedOn": {
-                        "type": "string",
-                        "description": "Timestamp in RFC3339 format (yyyy-MM-ddTHH:mm:ssZ) when the last message arrived at the server.",
-                        "x-ms-summary": "last Message Received On"
-                      },
-                      "deletedOn": {
-                        "type": "string",
-                        "description": "Timestamp in RFC3339 format (yyyy-MM-ddTHH:mm:ssZ) when the chat thread was deleted.",
-                        "x-ms-summary": "Deleted On"
-                      }
-                    }
-                  },
-                  "description": "Collection of chat threads.",
-                  "x-ms-summary": "Chat Threads"
-                },
-                "nextLink": {
-                  "type": "string",
-                  "description": "If there are more chat threads that can be retrieved, the next link will be populated.",
-                  "x-ms-summary": "Next Link"
+                    "$ref": "#/definitions/AgedBalance"
+                  }
                 }
               }
             }
           },
-          "401": {
-            "description": "Unauthorized"
-          },
-          "403": {
-            "description": "Forbidden"
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
           }
-        },
-        "summary": "List chat threads",
-        "description": "List chat threads.",
-        "operationId": "ListChatThreads",
```

Comment:
The operation has been deleted. This change breaks existing flows. ValuePath: operation/ListChatThreads

Reply:
We are adding a new template for a custom connector, so nothing is actually deleted.
Endpoint is deleted compared to prior iteration because prior iteration had incorrect swagger included.